qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -141,6 +142,11 @@ def setup(self):
         self.messagedisplay = LogMessageDisplay(self)
         self.messagedisplay.register_plugin()
 
+        self.set_splash("Loading IPython console")
+        from workbench.plugins.jupyterconsole import JupyerConsole
```

Comment:
Is the spelling of `JupyerConsole` intentional? Shouldn't it be `JupyterConsole`?

Reply:
Nope that's my poor spelling at work. It should be `Jupyter.` Thanks, I'll fix that.